onboard/catkin_ws/src/cv/scripts/detection.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+import rospy
+import os
+import yaml
+from custom_msgs.msg import Object
+from custom_msgs.srv import ToggleModel
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge
+from detecto.core import Model
+
+
+class Detector:
+
+    # Load in models and other misc. setup work
+    def __init__(self):
+        self.bridge = CvBridge()
+        self.camera = rospy.get_param('~/{}/camera'.format(rospy.get_name()))
```

Comment:
This isn't going to work if you use anonymous nodes, since there will be a large number after the node name. You would have to have the node name in the launch file as the camera param itself or something like that.